.github/workflows/macos_integration.yaml

```diff
@@ -0,0 +1,37 @@
+# TODO (polina-c): uncomment after figuring out https://github.com/flutter/flutter/issues/125636
```

Comment:
The test should not behave differently with -d flutter-tester and -d macos, so I suggest we delete this file all together.

Reply:
removed